content/en/th3/500_accessibility/311_screenReaders.md

```diff
@@ -7,28 +7,64 @@ weight: '502'
 
 ## Reading with a screen reader
 
-When launching Thorium, if a screen reader is detected, the relative
-functions are activated.
+### Activating screen reader support 
 
-Note that screen readers use a separate buffer to access the HTML
-content which results in the visual scroll / page offset to be out of
-sync. In other words, Thorium is generally not aware of the user\'s
+Thorium automatically detects assistive technology, but, since 3.3, users must explicitly activate screen reader support by ticking the checkbox in the application settings (this is a global preference for the main library window as well as all reader windows). 
+
+In the previous versions, this support was automatically activated. We had to change because of third party apps creating false positives because they activate assistive technology privileges.
+
+Thorium adapts its behavior when screen reader support is enabled:
+* Text input fields for bookshelf searches require explicit validation (e.g., pressing Enter or clicking the search button) to trigger the search, instead of the default "search as you type" behavior.
+* In the reader window, EPUB HTML documents default to scroll mode instead of paginated presentation to avoid issues with CSS columns.
```

Comment:
> EPUB HTML documents default to scroll mode instead of paginated presentation to avoid issues with CSS columns.

===>

> The presentation of EPUB HTML documents is forced to "scroll" mode instead of the default "paginated" view, in order to avoid issues with CSS columns.